docs/Modders/Creating-a-UI-skin.md

```diff
@@ -92,32 +118,34 @@ To create a config for your skin you just need to create a new .json file under
 This is an example of such a config file that will be explain below:
 
 ```json
+{
     "baseColor": {"r":1,"g":0,"b":0,"a":1},
     "skinVariants": {
         "MainMenuScreen/MenuButton": {
-          "image": "MyCoolNewDesign",
+            "image": "MyCoolNewDesign"
         },
         "TechPickerScreen/TechButton": {
-          "image": "MyCoolNewDesign",
-          "alpha": 0.7
+            "image": "MyCoolNewDesign",
+            "alpha": 0.7
         },
         "WorldScreen/TopBar/ResourceTable": {
-          "alpha": 0.8
+            "alpha": 0.8
         },
         "WorldScreen/UnitTable": {
-          "tint": {"r": 1, "g": 0, "b": 0},
-          "image": "WorldScreen/TopBar/ResourceTable",
-          "alpha": 0.4
+            "tint": {"r": 1, "g": 0, "b": 0},
+            "image": "WorldScreen/TopBar/ResourceTable",
+            "alpha": 0.4
         },
         "WorldScreen/Minimap/Background": {
-          "tint": {"r": 0.2, "g": 0.4, "b": 0.45, "a": 1}
-        },
+            "tint": {"r": 0.2, "g": 0.4, "b": 0.45, "a": 1}
+        }
     }
+}
```

Comment:
Can we have the colors be regular RGB 0-255 values? 'normalizing' values is not exactly natural when dealing in colors

Reply:
The libgdx Color class does not support integer RGB values for deserialization so I don't think it will be particularly easy to do so 
But yea I don't like working with these float values either

Reply:
I have a helper function fir converting lists of int [1,2,3] to Colors, I use it for all moddable objects